### DISCUSSION TITLE
Allow attaching descriptions to the playlist lobbies

### DISCUSSION THREAD

**Discussion by venix12** [ppy/osu#17221] [Ideas] [answered]
Would be a nice addition for spotlights lobbies since such kind of description could contain a brief information on how spotlights system work along with a link to wiki or newspost for more of an in-depth explaination of those stuffs (mostly for people randomly finding out spotlighted lobbies while playing around with lazer). 

Would also likely be useful for regular lobbies to explain the gimmick or purpose of a playlist.

**Comment by frenzibyte** [comment, marked as answer]
Tracked in #17224
